plugins/sudoers/iolog_client.c

```diff
@@ -166,6 +166,20 @@ connect_server(const char *host, const char *port, struct timespec *timo,
 	    sock = -1;
 	    continue;
 	}
+
+    if (tcp_keepalive) {
+        int keepalive = 1;
+        if (setsockopt(sock, SOL_SOCKET, SO_KEEPALIVE, &keepalive,
```

Comment:
Please compare to -1 instead of <0 here.

Reply:
fixed